src/mesh-analysis.rs

```diff
@@ -0,0 +1,17 @@
+use nalgebra as na;
```

Comment:
This file is not part of the compilation and in fact contains multiple compilation errors (from what I see). In rust, not all files are compiled, but only those referenced in the project by `mod` declarations.

There are multiple problems here:

1) The filename should be `mesh_analysis.rs`, not `mesh-analysis.rs`, because `-` is not a valid identifier character in rust and modules are not string tokens, but identifier tokens. 
2) You need to add a `mod mesh_analysis;` to `lib.rs` (search for `mod` in the code to find the section to put it in)